native/cocos/platform/android/AndroidPlatform.cpp

```diff
@@ -480,6 +497,14 @@ void gameControllerStatusCallback(const int32_t controllerIndex,
 AndroidPlatform::~AndroidPlatform() = default;
 
 int AndroidPlatform::init() {
+#if CC_USE_XR
+    registerInterface(std::make_shared<XRInterface>());
+#endif
+    IXRInterface *xr = getInterface<IXRInterface>();
```

Comment:
Unified use of one interface？ “getInterface<IXRInterface>()" or "CC_GET_XR_INTERFACE"?

Reply:
Change to "CC_GET_XR_INTERFACE"